src/order-groups.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is additionally under an MPL-2.0 license. See LICENSE.note.
```

Comment:
I think this needs to be a little more specific. You should repeat that that copyright holders are the data table team (plus now RStudio for your contributions), and be clear about the original source.